Sources/SwiftDocC/Model/Rendering/RenderContentCompiler.swift

```diff
@@ -207,6 +205,44 @@ struct RenderContentCompiler: MarkupVisitor {
         return [RenderInlineContent.strikethrough(inlineContent: strikethrough.children.reduce(into: [], { result, child in result.append(contentsOf: visit(child))}) as! [RenderInlineContent])]
     }
 
+    mutating func visitBlockDirective(_ blockDirective: BlockDirective) -> [RenderContent] {
+        switch blockDirective.name {
+        case Snippet.directiveName:
+            guard let snippetURL = blockDirective.arguments()[Snippet.Semantics.Path.argumentName],
+                  let snippetReference = resolveSymbolReference(destination: snippetURL.value),
+                  let snippetEntity = try? context.entity(with: snippetReference),
+                  let snippetSymbol = snippetEntity.symbol,
+                  let snippetMixin = snippetSymbol.mixins[SymbolGraph.Symbol.Snippet.mixinKey] as? SymbolGraph.Symbol.Snippet else {
+                return []
+            }
+
+            let docCommentContent = snippetEntity.markup.children.flatMap { self.visit($0) }
```

Comment:
Have I understood correctly that this comes from the snippet's doc comment? If so, does that mean that snippets support styled inline content (e.g. strong and emphasis)?

If that's supported, would it be worth adding some styling to the test data?

Reply:
That's right, you can write styled markdown in those comments. I'm not convinced that it needs its own test case since it's just a call to `visit` and those methods are already tested independently, but let me know if you feel strongly about it.